/aruco_node:
    ros__parameters:
        # file
        output_dir: ./dataset
        # Input topics
        image_topic: /camera/camera/color/image_raw # Input image topic
        use_depth_input: True # Use depth image for 3D pose estimation
        depth_image_topic: /camera/camera/aligned_depth_to_color/image_raw # Input depth image topic
        camera_info_topic: /camera/camera/aligned_depth_to_color/camera_info  # Input camera info topic with camera intrinsic parameters
        camera_frame: camera_color_optical_frame # Camera link frame of reference
        # Output topics
        output_image_topic: /aruco/image # Output topic with visualization of detected markers drawn on image
